lib/mi_bridges/driver.rb

```diff
@@ -89,7 +89,6 @@ def run
     attr_reader :snap_application
 
     def setup
-      DriverApplication.destroy_all
```

Comment:
@jayroh / @jessieay - This in particular worried me a bit; I'm assumin we only want to be deleting driver applications in particular cases, not every time we drive?

Reply:
you need to have this line if you are going to re-drive for the same snap app multiple times (which is what we are doing now).

We will want to remove for prod, though, so maybe put it in an env check in the meantime? or remove scoped to the id of the current snap app?

Reply:
This *could* remain if we scope it to Rails.env.development only - because (at least for NOW) we only want to run the driver for brand new applications. Otherwise this would pick up previous runs.

Reply:
I moved it into the `bin/drive` script; but I like the notion of renaming `bin/drive` to `bin/dev_drive` and `bin/demo_drive` with the default environment variables set for that.